torchgeo/trainers/classification.py

```diff
@@ -90,9 +97,13 @@ def __init__(self, **kwargs: Any) -> None:
             in_channels: Number of input channels to model
             learning_rate: Learning rate for optimizer
             learning_rate_schedule_patience: Patience for learning rate scheduler
+            freeze_backbone: Fine-tune the cls head by freezing the backbone
```

Comment:
Fine-tune the classifier head. Also, fine-tune is misleading since this is only used in the case where you aren't fine-tuning the model, you're doing linear probing.